apps/bttester/src/btp_gap.c

```diff
@@ -2000,6 +2001,47 @@ set_filter_accept_list(const void *cmd, uint16_t cmd_len,
     return BTP_STATUS_SUCCESS;
 }
 
+#if MYNEWT_VAL(BLE_EXT_ADV)
+static uint8_t
+set_ext_advertising(const void *cmd, uint16_t cmd_len,
+                    void *rsp, uint16_t *rsp_len)
+{
```

Comment:
This should only set this in current settings and set adv_param->legacy_pdu accordingly

Also, if advertising is already enabled this may simply fail,  if ext_adv is not enabled in syscfg it should also fail.  


so in short:
```
if (cp->setting) {
   current_settings |= BIT(BTP_GAP_SETTINGS_EXTENDED_ADVERTISING); 
   adv_param->legacy_pdu = 0;
} else {
   current_settings &= ~BIT(BTP_GAP_SETTINGS_EXTENDED_ADVERTISING);
   adv_param->legacy_pdu = 1;
}

``

Reply:
Done